diff --git a/node_modules/typeorm/driver/react-native/ReactNativeDriver.js b/node_modules/typeorm/driver/react-native/ReactNativeDriver.js
index fd228cb..909cf48 100644
--- a/node_modules/typeorm/driver/react-native/ReactNativeDriver.js
+++ b/node_modules/typeorm/driver/react-native/ReactNativeDriver.js
@@ -5,7 +5,6 @@ var AbstractSqliteDriver_1 = require("../sqlite-abstract/AbstractSqliteDriver");
 var ReactNativeQueryRunner_1 = require("./ReactNativeQueryRunner");
 var DriverOptionNotSetError_1 = require("../../error/DriverOptionNotSetError");
 var DriverPackageNotInstalledError_1 = require("../../error/DriverPackageNotInstalledError");
-var PlatformTools_1 = require("../../platform/PlatformTools");
 var ReactNativeDriver = /** @class */ (function (_super) {
     tslib_1.__extends(ReactNativeDriver, _super);
     // -------------------------------------------------------------------------
@@ -80,7 +79,10 @@ var ReactNativeDriver = /** @class */ (function (_super) {
      */
     ReactNativeDriver.prototype.loadDependencies = function () {
         try {
-            this.sqlite = PlatformTools_1.PlatformTools.load("react-native-sqlite-storage");
+            console.log('-------------------------------------------')
+            console.log('requiring!!!')
+            console.log('-------------------------------------------')
+            this.sqlite = require("react-native-sqlite-storage");
         }
         catch (e) {
             throw new DriverPackageNotInstalledError_1.DriverPackageNotInstalledError("React-Native", "react-native-sqlite-storage");
